eng/testing/tests.wasm.targets

```diff
@@ -15,10 +15,7 @@
         But we do want to set it for Configuration=Debug .
     -->
     <WasmDebugLevel Condition="'$(Configuration)' == 'Debug' and '$(WasmDebugLevel)' == ''">-1</WasmDebugLevel>
-    <!-- We want to have WasmDebugLevel=0, but if DebuggerSupport=true then BrowserWasmApp.targets
-         *will* override it. So, set this to "reset-to-zero" here, and reset the value later
-         in targets. -->
-    <WasmDebugLevel Condition="'$(DebuggerSupport)' == 'true' and '$(ContinuousIntegrationBuild)' == 'true' and '$(WasmDebugLevel)' == ''">reset-to-zero</WasmDebugLevel>
+    <WasmDebugLevel Condition="'$(ContinuousIntegrationBuild)' == 'true' and '$(WasmDebugLevel)' == ''">0</WasmDebugLevel>
```

Comment:
why do we only do this for ContinuousIntegrationBuild ?

Reply:
If you are running it on your dev machine, you probably could attach debugger at some point.

Reply:
So locally it will still take 70mins?

Reply:
What I'm getting at: anytime we have configuration that differs between CI and local it's a pain to investigate so we should make sure it is really what we want

Reply:
I don't have strong opinion

Reply:
I kind of do :) if someone wants to debug locally they can set WasmDebugLevel right?